src/monitoring_service/cli.py

```diff
@@ -129,7 +156,8 @@ def main(
         required_confirmations=confirmations,
         db_filename=state_db,
     )
-
+    while gevent.spawn(check_gas_reserve(web3, private_key)) is True:
```

Comment:
This looks really strange. Are you sure it does what you intent?

Reply:
done